libdnf/rpm/solv/goal_private.cpp

```diff
@@ -144,6 +142,11 @@ bool GoalPrivate::resolve() {
         solver_set_flag(libsolv_solver, SOLVER_FLAG_ALLOW_DOWNGRADE, 1);
     }
 
+    // Set up vendor locking modes
```

Comment:
Please could you add TODO? There is no mechanism how to modify it according to configuration in conf.

Reply:
Oh, I thought there already was because I saw the config option already exist. Is there an example of how I can wire that up?

Reply:
There are two ways 

1. Required value in constructor
2. To create a setter for GoalPrivate class. Then the value will be set from configuration.

I suggest that the second option would be a better one. Or just add TODO  and it will be implemented later on. No problem with that.

Reply:
Didn't I already make it a value for the constructor?

Reply:
I went ahead and added a TODO comment, I don't have anymore time to work on this at the moment.